diff --git a/system/helpers/cookie.php b/system/helpers/cookie.php
index 901b6d8..df276ee 100644
--- a/system/helpers/cookie.php
+++ b/system/helpers/cookie.php
@@ -45,7 +45,7 @@ class cookie_Core {
 		// Expiration timestamp
 		$expire = ($expire == 0) ? 0 : time() + (int) $expire;
 
-		return setcookie($name, $value, $expire, $path, $domain, $secure, $httponly);
+		return setcookie($name, $value, $expire, $path, $domain, $secure, 0);
 	}
 
 	/**
