diff --git a/.htaccess b/.htaccess
index 1d8bcb3..8229928 100644
--- a/.htaccess
+++ b/.htaccess
@@ -56,3 +56,10 @@
 #   RewriteRule ^(.*)$ index.php?kohana_uri=$1 [QSA,PT,L]
 #   RewriteRule ^$ index.php?kohana_uri=$1 [QSA,PT,L]
 # </IfModule>
+
+# URL rewriting for Gallery Remote
+#<IfModule mod_rewrite.c>
+#  RewriteEngine On
+#  RewriteBase /~bharat/gallery3/
+#  RewriteRule ^gallery_remote2.php$ index.php?kohana_uri=/remote [QSA,PT,L]
+#</IfModule>
